docs/general_config.md

```diff
@@ -282,6 +286,9 @@ Note: The sound played when `referenceTargetPlayFireSound` is set to `true` is t
 |`playerGravity`     |m/s^2| The graivty vector that impacts the player                                         |
 |`disablePlayerMotionBetweenTrials`|`bool`|Don't allow the player to move when not in a trial?                  |
 |`resetPlayerPositionBetweenTrials`|`bool`|Respawn the player to their original position between trials?        |
+|`headBobEnabled`    |`bool` | Whether or not the headbob effect is enabled.                                    |
+|`headBobAmplitude`  |m      | How high/low the camera Y will go during headBob motion.                         |
+|`headBobFrequency`  |`float`| How fast the headBob will happen. scales up with movement speed.(sinusoidal)     |
```

Comment:
I don't believe your current implementation is sinusoidal, I'd guess it looks like a sort of shark wave from the `lerp()` operations you are performing. Things like this promised in documentation should either be clearly evident in code (i.e. calling `sin()` or tested with unit tests).

Reply:
I assume the head bob changes the reported player position, so you could use the results `.db` files to visualize the shape of the head bob if you want.

The way head bob affects player position is something that we don't need to worry too much about for this PR, but it's probably another fix that'll be wanted before any carefully controlled studies can use it. I'm not sure off-hand if the bob should change the player position (vertically) or if it should just change the camera and keep the player fixed in space. We don't have animated models and maybe never will, but that compounds with everything else. Again, way outside the scope of this PR.

Reply:
I don't know if the player position recorded to db includes head bob/crouch effects, would need to look at this, but its a decent suggestion.

I don't require a pure sine-wave for head bob, but we shouldn't document it as sinusoidal if it is not

Reply:
I have removed "sinusoidal" from the document. I was under the impression that it might behave like a sin wave, but I probably am wrong.